src/components/Albums/AlbumForm.vue

```diff
@@ -312,4 +313,12 @@ export default {
 .left-buttons {
 	flex-grow: 1;
 }
+
+@media (max-width: 1280px) {
```

Comment:
To be a bit more consistent with other places in Photos and 1024 breakpoint.

```suggestion
@media only screen and (max-width: 1020px)
```

Reply:
```suggestion
@media (max-width: 1024px) {
```